satpy/scene.py

```diff
@@ -101,11 +101,15 @@ def __init__(self, filenames=None, reader=None, filter_parameters=None,
         will be searched for a Reader that can support the provided files. This
         can take a considerable amount of time so it is recommended that
         ``reader`` always be provided. Note without ``filenames`` the Scene is
-        created with no Readers available requiring Datasets to be added
-        manually::
+        created with no Readers available.  When a Scene is created with no Readers,
+        each xarray.DataArray must be added manually::
 
             scn = Scene()
-            scn['my_dataset'] = Dataset(my_data_array, **my_info)
+            scn['my_dataset'] = DataArray(my_data_array, attrs={})
+
+        The `attrs` dictionary contains the metadata for the data. Some important keys include the
+        platform_name, sensor, name, wavelength, units, calibration, standard_name, start_time,
+        and the area which contains the projection information (AreaDefinition or SwathDefinition) for this data. 
```

Comment:
```suggestion
        The `attrs` dictionary contains the metadata for the data. See
        :ref:`dataset_metadata` for more information.
```

I'm not sure if that reference name is correct.